Tests.Integration.Presentation.Web/Tools/Internal/ItSystem/ItSystemInternalV2Helper.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using System;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V2.Internal.Request.Options;
+
+namespace Tests.Integration.Presentation.Web.Tools.Internal.ItSystem
+{
+    public static class ItSystemInternalV2Helper
+    {
+        public static async Task<HttpResponseMessage> GetLocalOptionTypes(Guid organizationUuid, string choiceTypeName)
+        {
+            var cookie = await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            return await HttpApi.GetWithCookieAsync(
+                TestEnvironment.CreateUrl($"api/v2/internal/it-systems/{organizationUuid}/local-option-types/{choiceTypeName}"), cookie);
```

Comment:
since each method is prefixed by ""api/v2/internal/it-systems/{organizationUuid}/local-option-types/{choiceTypeName}", consider making a private helper method (like GetRoutePrefix) that creates this string for each helper method.